cms_form/models/cms_search_form.py

```diff
@@ -24,6 +24,11 @@ class CMSFormSearch(models.AbstractModel):
     _form_results_orderby = ''
     # declare fields that must be searched w/ multiple values
     _form_search_fields_multi = ()
+    # declare custom domain computation rules
+    _form_search_domain_rules = {
+        # field name: (leaf field name, operator, format value),
+        # 'product_id': ('product_id.name', 'ilike', '{}'),
```

Comment:
is that a usage example? :slightly_smiling_face:

Reply:
yes, don't complain! :D